### ISSUE TITLE
Build script fails through image crate

### ISSUE THREAD

**Issue by FlareFlo** [rust-av/dav1d-rs#81]
Building image with the `avif` feature fails with [this output
](https://pastebin.com/EXrqL9J1)
Building inside an isolated container works however:
```dockerfile
FROM archlinux
RUN pacman -Syu --noconfirm rustup base-devel nasm
RUN pacman --noconfirm -Scc
RUN rustup install stable --profile minimal
RUN cargo new broken_avif
WORKDIR broken_avif
RUN cargo add image --features avif --no-default-features
RUN cargo build
```
Dav1d is installed on the system with [version 1.3.0-1](https://archlinux.org/packages/extra/x86_64/dav1d/).
This could be related to #80, but i am not quite sure.

**Comment by sdroege** [comment]
```
error: failed to run custom build command for `dav1d-sys v0.3.5`
```

That's a quite ancient version. The build will work with newer versions but note https://github.com/rust-av/dav1d-rs/pull/83: dav1d 1.3.0 breaks API/ABI, so either you have to downgrade to an older version or wait for the next release of the Rust bindings (which will require 1.3.0 or newer).

**Comment by FlareFlo** [comment]
I figured something along those lines was going on. Looking forward to the new version!
Feel free to close this issue, I am not quite sure if this can/should be considered solved already.

**Comment by FlareFlo** [comment]
Fixed in 3a5c34b8e830c2f9eed71e965f9f2c6ba1490776
